org.eclipse.xtext.ui/src/org/eclipse/xtext/ui/PluginImageHelper.java

```diff
@@ -184,4 +193,26 @@ public void bundleChanged(BundleEvent event) {
 		}
 	}
 
+	/** Find the descriptor of the image with the given id.
+	 *
+	 * @param name the identifier of the image. It may be qualified with the plugin's id.
+	 * @return the descriptor.
+	 * @since 2.14
+	 */
+	protected ImageDescriptor findImage(String name) {
```

Comment:
what is the name you are looking for? how do you encode the image name

Reply:
pluginname.imagename.extension?

Reply:
The supported format is of name is "<pluginid>.<imagename>".

Reply:
Can you please make this a bit more explicit in the javadoc with an example

Reply:
I suggest to use this syntax for cross-bundle lookup:
`/bundlename/path/image.name.with.dot.ext`
That is, a leading slash and a bundleName, and slash-delimited path to the resource of that bundle.

Reply:
We could also use the standard Eclipse RCP notation: `platform:/plugin/<bundlename>/<path>/<image.name.with.dot.ext>`

What is your preferred one?

Reply:
I‘d prefer platform plugin URIs. 
@szarnekow @cdietrich WDYT?

Reply:
I don't have a strong preference. Most importantly it has to be unambiguous. Testing for a '/' as the first char and putting `platform:/plugin` in front of it are requiring the complete URI is both ok to me.

Reply:
I have use the platform-uri syntax.